index.html

```diff
@@ -1,229 +1,300 @@
+                  </li>
+  
+                  <li class="ellipse">
+                    <img src="./images/Ellipse 1 (1).png" alt="HTML logo">
+                    <p>HTML</p>
+                  </li>
+  
+                  <li class="ellipse">
+                    <img src="./images/Ellipse 1 (2).png" alt="CSS logo">
+                    <p>CSS</p>
+                  </li>
+                </ul>
               </li>
               <li>
-                <img src="assets/image/htmlLogo.png" alt="Html" />
-                <a href="#">HTML</a>
+                <div class="frame">
+                  <p class="frame-title" >Frameworks</p>
+                  <button class="lang-btn"><i class="fa-solid fa-2x fa-angle-right"></i></button>
+                </div>
+                <hr>
               </li>
               <li>
-                <img src="assets/image/css_logo.png" alt="CSS" />
-                <a href="#">CSS</a>
+                <div class="frame">
+                  <p class="skill-title">Skills</p>
+                  <button class="lang-btn"><i class="fa-solid fa-2x fa-angle-right"></i></button>
+                </div>
               </li>
+              <hr>
             </ul>
-          </li>
-          <li class="lang-exp">
-            <div class="no-1">
-              <p>Frameworks</p>
-              <img src="assets/image/arrow_bottom.png" alt="arrow_bottom" />
-            </div>
-            <hr />
-          </li>
-
-          <li class="lang-exp">
-            <div class="no-1">
-              <p>Skills</p>
-              <img src="assets/image/arrow_bottom.png" alt="arrow_bottom1" />
-            </div>
-            <hr />
-          </li>
-        </ul>
-      </div>
-    </div>
-  </section>
-</body>
-
+              <!-------------------------------------- Contact form------------------------------->
+            <div class="about-contact">
+              <div class="title-container">
+                <h2 class="contact">Contact Me</h2>
+                
+      <p class="p-pargraph">
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it
+      </p>
+              <div class="form-container">
+                <form action="https://formspree.io/f/mqkoyoby" method="POST" class="labels">
+                    <input type="text" placeholder="Yeremias" name="name" id="name_form" maxlength="30" required />
+                    <input type="email" placeholder="natajayanj14@gmail.com" name="email" maxlength="500" id="email_form" />
+                    <textarea name="message" placeholder="Write your message here" id="message" cols="30" rows="10"
+                    maxlength="500" required></textarea>
+                  <button id="get-submit" type="submit" class="beg-btn">Get in touch</button>
+        
+                </form>
+              </div>
+              <hr>
+            </div>     
+          </div>
```

Comment:
- [ ] As mentioned by the previous reviewer :point_up: please stick to the Figma design as much as possible but your design is still not the same as the Figma design kindly fix it :+1:  check below :point_down:  the comparison between your design and Figma design :slightly_smiling_face: 

**Your Design**
![Screenshot from 2023-04-05 16-06-49](https://user-images.githubusercontent.com/45226848/230064517-c2e3b699-0867-4694-a03f-e09f65bd9804.png)

**Figma design**  
![Screenshot from 2023-04-05 16-08-24](https://user-images.githubusercontent.com/45226848/230064544-a6a023e1-0af5-4675-b740-a919e9f6f8c7.png)

- [ ] If you see your design the button is in the centre but in Figma design, the button is on the lift side so kindly fix it  :+1: 
- [ ] Also, if you check your design the image is not shown clearly kindly fix this issue :slightly_smiling_face: